vscode-antimony/all-requirements.txt

```diff
@@ -15,4 +15,5 @@ requests-cache==0.5.2
 six==1.15.0

```

Comment:
Is there a reason for specifying a specific release? Why not a release at a minimal level? My concern is that the package may have dependency conflicts if the user has other packages with overlapping dependencies.

Reply:
Yes, several packages tend to change and break our extension so after a few experiments, I found that the best way is to specify a specific version.
This wouldn't be a problem on the user's end because we're packaging dependencies when deploying and not using anything from the user's environment.